src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -294,56 +294,18 @@ void InitSurfaceHeatBalance(EnergyPlusData &state)
 
     SetSurfaceWindSpeedAt(state);
     SetSurfaceWindDirAt(state);
+    InitSurfacePropertyViewFactors(state);
```

Comment:
New function `InitSurfacePropertyViewFactors(state)` replaced the deleted code below.